### DISCUSSION TITLE
Dont tell me my password is weak

### DISCUSSION THREAD

**Discussion by stayupthetree** [louislam/dockge#207] [Feature Request]
Title. Let me handle my own security, thanks.

**Comment by ApricotFarm** [comment]
I second this enthusiastically, although I'd like to express it a bit more politely. It's a fairly major inconvenience to be **_required_** to have final-product-level security when one is just experimenting with a new package, etc. I do see that you've implemented the option to defeat authentication altogether, which is great, but I'd like to see a bit more nuance to it. I actually think Portainer (of all things) gets this right. The ability to tailor one's security policies is really helpful, and with reasonable default settings in place, I think is perfectly responsible in terms of propagating security.
